src/actors/symbolication.rs

```diff
@@ -2189,6 +2189,53 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn test_wasm_payload() -> anyhow::Result<()> {
+        let (service, _cache_dir) = setup_service();
+        let (_symsrv, source) = test::symbol_server();
+
+        let modules: Vec<RawObjectInfo> = serde_json::from_str(
```

Comment:
Deserializing directly into `CompleteObjectInfo` would require us to specify a set of irrelevant attributes and would no longer match the input to normal requests. 

I will revisit test utilities for this when porting the actual E2E tests.